src/core.spec.js

```diff
@@ -0,0 +1,12 @@
+/* global beforeEach, describe, expect, it */
```

Comment:
Yeah, it's more pod-like which is what we're moving to in other repos as well. Glob syntax makes filtering these files out on build steps really trivial and putting the tests next to their source keeps it top of mind.